docs/tutorials/controlling-wedo2-motor.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Controlling a WeDo 2.0 motor
+subject: Networking
```

Comment:
Yes, it might be better. I choosed Networking because most of the work is around bluetooth and also because I expect to extend the tutorial with sensor reading but I can make also an anothe tutorial just for for that.